hubploy/helm.py

```diff
@@ -1,4 +1,20 @@
 """
+Convention based helm deploys
+
+Expects the following configuration layout from cwd:
+
+chart-name/ (Helm deployment chart)
+deployments/
+  - deployment-name
+    - image/ (optional)
+    - secrets/
+      - prod.yaml
+      - staging.yaml
+    - config/
+      - common.yaml
+      - staging.yaml
+      - prod.yaml
+
```

Comment:
I'm slightly conflicted about re-documenting this structure here. It is helpful, when its kept up to date, but keeping it up to date when its written out in multiple places is a bit messy. Even in this file, its written out on line 110 for example.

I think it's fine to repeat ourselves a bit though, its probably better than assuming its fresh in the reader's mind.